backend/tests/conftest.py

```
@@ -1,36 +1,40 @@
-
+        session.close()
+        transaction.rollback()
+        connection.close()
 
-@pytest.fixture(scope="function")
+@pytest.fixture
 def client(db_session):
-    """Provides a TestClient instance with the test database."""
-    from fastapi.testclient import TestClient
-    from app.main import app
+    def override_get_db():
```

Ruff (0.8.2)

Missing return type annotation for private function `override_get_db`

(ANN202)